Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -30200,4 +30250,344 @@
+        <LifeMax value="150"/>
+        <LifeArmorName value="Unit/LifeArmorName/ProtossPlating"/>
+        <ShieldsStart value="100"/>
+        <ShieldsMax value="100"/>
+        <ShieldRegenDelay value="10"/>
+        <ShieldRegenRate value="2"/>
+        <ShieldArmorName value="Unit/ShieldArmorName/ProtossPlasmaShields"/>
+        <Mover value="Fly"/>
+        <Speed value="2.8125"/>
+        <Acceleration value="1.875"/>
+        <StationaryTurningRate value="539.4726"/>
+        <TurningRate value="539.4726"/>
+        <Sight value="9"/>
+        <Height value="3.75"/>
+        <VisionHeight value="4"/>
+        <Food value="-3"/>
+        <CostCategory value="Army"/>
+        <CostResource index="Minerals" value="275"/>
+        <CostResource index="Vespene" value="125"/>
+        <AttackTargetPriority value="20"/>
+        <DamageDealtXP value="1"/>
+        <DamageTakenXP value="1"/>
+        <KillXP value="80"/>
+        <AbilArray Link="stop"/>
+        <AbilArray Link="attack"/>
+        <AbilArray Link="move"/>
+        <AbilArray Link="Warpable"/>
+        <AbilArray Link="ProgressRally"/>
+        <AbilArray Link="AP_ScoutPurifierCoronaBeam"/>
+        <BehaviorArray Link="AP_SOAHeroicShieldProcBuff"/>
+        <BehaviorArray Link="AP_ScoutPurifierSolariteReactor"/>
+        <BehaviorArray Link="AP_WarpInEffect"/>
+        <WeaponArray Link="AP_ScoutPurifierAir"/>
+        <WeaponArray Link="AP_ScoutPurifierGround"/>
+        <CardLayouts>
+            <LayoutButtons Face="Move" Type="AbilCmd" AbilCmd="move,Move" Row="0" Column="0"/>
+            <LayoutButtons Face="Stop" Type="AbilCmd" AbilCmd="stop,Stop" Row="0" Column="1"/>
+            <LayoutButtons Face="MoveHoldPosition" Type="AbilCmd" AbilCmd="move,HoldPos" Row="0" Column="2"/>
+            <LayoutButtons Face="Attack" Type="AbilCmd" AbilCmd="attack,Execute" Row="0" Column="4"/>
+            <LayoutButtons Face="MovePatrol" Type="AbilCmd" AbilCmd="move,Patrol" Row="0" Column="3"/>
+            <LayoutButtons Face="AP_ScoutPurifierWeaponRange" Type="Passive" Requirements="AP_HaveScoutWeaponRange" Row="1" Column="0"/>
+            <LayoutButtons Face="AP_ScoutGraviticThrusters" Type="Passive" Requirements="AP_HaveScoutSCBWSpeedUpgrade" Row="1" Column="1"/>
+            <LayoutButtons Face="AP_ScoutPurifierSideMissiles" Type="Passive" Requirements="AP_ScoutPurifierHaveSideMissiles" Row="1" Column="2"/>
+            <LayoutButtons Face="AP_ScoutPurifierStructureTargeting" Type="Passive" Requirements="AP_ScoutPurifierHaveStructureTargeting" Row="1" Column="3"/>
+            <LayoutButtons Face="Rally" Type="AbilCmd" AbilCmd="ProgressRally,Rally1" Row="2" Column="4"/>
+            <LayoutButtons Face="AP_ScoutPurifierCoronaBeam" Type="AbilCmd" AbilCmd="AP_ScoutPurifierCoronaBeam,Execute" Row="2" Column="0"/>
+            <LayoutButtons Face="AP_ScoutPurifierSolariteReactor" Type="Passive" Requirements="AP_ScoutPurifierHaveSolariteReactor" Row="1" Column="4"/>

```

Comment:
Add the cooldown behavior to this (like Ultras have on revive) in order to display the remaining cooldown duration of the effect on the icon